app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -699,6 +771,10 @@ class HomeActivityTest {
     ).check(matches(withText(stringToMatch)))
   }
 
+  private fun verifyNumberOfGridColumns(columnCount: Int) {
```

Comment:
Perhaps: ``verifyHomeRecyclerViewHasGridColumnCount`` might be clearer.

Reply:
Done